packages/react/src/input-base/InputBase.css.ts

```diff
@@ -63,10 +58,11 @@ export const input = recipe({
     {
       bg: "transparent",
       flex: "auto",
+      fontWeight: "400",
     },
     style({
-      fontSize: "16px",
-      lineHeight: "22px",
```

Comment:
we need to revert this change - and we can add a comment explaining it - line-height was set to `22px` to adjust for the `1px` border on top and bottom